Gemfile

```diff
@@ -43,7 +43,7 @@ gem "jquery-ui-rails"
 gem "simple_form", '~> 3.1.0'
 gem 'nokogiri'
 gem 'equivalent-xml'
-gem 'solr_wrapper'
+gem 'solr_wrapper', '0.7.3'
```

Comment:
The latest version switched to SOLR 6. This was a breaking change for GeoCon. Let me see if I can track down the issues.